src/components/Collection/Series/EditSeriesTabs/FileActionsTab.tsx

```diff
@@ -10,6 +16,10 @@ type Props = {
 const FileActionsTab = ({ seriesId }: Props) => {
   const { mutate: rehashSeriesFiles } = useRehashSeriesFilesMutation(seriesId);
   const { mutate: rescanSeriesFiles } = useRescanSeriesFilesMutation(seriesId);
+  const { mutate: relocateSeriesFiles } = useRelocateSeriesFilesMutation(seriesId);
+  const unsuported = () => {
```

Comment:
typo, but also maybe name it better
```suggestion
  const showUnsupportedToast = () => {
```